arangod/Replication/DatabaseInitialSyncer.h

```diff
@@ -192,11 +193,24 @@ class DatabaseInitialSyncer final : public InitialSyncer {
   Result fetchCollectionDump(arangodb::LogicalCollection*,
                              std::string const& leaderColl, TRI_voc_tick_t);
 
-  /// @brief incrementally fetch data from a collection
+  /// @brief incrementally fetch data from a collection using keys as the
+  /// primary document identifier
   // TODO worker safety
   Result fetchCollectionSync(arangodb::LogicalCollection*,
                              std::string const& leaderColl, TRI_voc_tick_t);
 
+  /// @brief incrementally fetch data from a collection using keys as the
+  /// primary document identifier
+  // TODO worker safety
```

Comment:
Something missing here?

Reply:
The TODO is just replicated from the original method, since any concerns about worker safety have not been addressed by this PR.